test/tools/ossfuzz/config/solidity.dict

```diff
@@ -40,7 +40,7 @@
 ".creationCode"
 ".delegatecall("
 ".gas("
-".kill("
```

Comment:
@bshastry can you check this?

Reply:
Question: Is the issue that using "kill" is illegal? Or is it that we should discourage developers from using it? If it's the latter I don't see why we should remove it from the dictionary list. The dictionary list is meant to make it easier for the fuzzer to synthesize corner cases, so as long as "kill" is legal, it is fine to keep it.

Reply:
Updated at 4476625456bce83aee1cab31c2d274bd65c0ec3d